src/test/java/catalinaTema5/DemoqaElements/WebTabels.java

```diff
@@ -0,0 +1,46 @@
+package catalinaTema5.DemoqaElements;
```

Comment:
Don't forget to rename also the file name, not only the class name to WebTables. Thanks!